aydin/nn/models/utils/n2s_dataset.py

```diff
@@ -0,0 +1,73 @@
+import numpy
+import torch
+from torch.utils.data import Dataset
+
+
+class N2SDataset(Dataset):
```

Comment:
I think it's a good idea to involve masking inside a Dataset object.

Reply:
I will make separate dataset for different masking approaches and names will reflect accordingly.